public_html/wp-content/mu-plugins/blocks/assets/src/speakers/block-content.scss

```diff
@@ -19,6 +19,7 @@
 	}
 
 	.wordcamp-speaker:after {
+		/* why not use the existing wp-clearfix class? */
```

Comment:
`wp-clearfix` isn't available on the front end. Currently we're loading this stylesheet on both the WP Admin and the front end so that the layout options work correctly regardless of theme.

Reply:
Ah, good point. I wonder if we should have our own `wordcamp-clearfix`, or maybe a duplicate `wp-clearfix`?

At the very least, it seems like it'd be good to have a rule with `clearfix` in the name, so its purpose would be obvious, and we could reuse it across blocks.

Reply:
Good call, I like the idea of adding a rule for `wordcamp-clearfix`.